<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TiltTracker</class>
 <widget class="QWidget" name="TiltTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="title">
     <property name="text">
      <string>&lt;h3&gt;Stylus Tilt Tracker&lt;/h3&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="description">
     <property name="text">
      <string>TODO description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TiltIndicator" name="tiltIndicator" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="tiltY">
       <property name="text">
        <string>&lt;h1&gt;0.00&lt;/h1&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="tiltX">
       <property name="text">
        <string>&lt;h1&gt;0.00&lt;/h1&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="tiltXLabel">
       <property name="text">
        <string>X Tilt</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="tiltYLabel">
       <property name="text">
        <string>Y Tilt</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TiltIndicator</class>
   <extends>QWidget</extends>
   <header>tiltindicator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
